mutable_tree.go

```diff
@@ -45,6 +46,18 @@ func (tree *MutableTree) VersionExists(version int64) bool {
 	return tree.versions[version]
 }
 
+// AvailableVersions returns all available versions
+func (tree *MutableTree) AvailableVersions() []int {
```

Comment:
It would be great to have a little test for `AvailableVersions`. I can add it it you do not have time.

Reply:
I will look to do it a well